webpack/assets/javascripts/react_app/components/toastNotifications/index.js

```diff
@@ -0,0 +1,30 @@
+import React, { Component } from 'react';
+import Toast from './toastListitem/';
+import * as ToastActions from '../../redux/actions/toasts';
+import { connect } from 'react-redux';
+import { map } from 'lodash';
+
+class ToastsList extends Component {
+
+  render() {
+    const { messages, deleteToast } = this.props;
+
+    return (
+      <div className="toast-notifications-list-pf">
+        {
+          map(
```

Comment:
@tbrisker this will actually not work....
See
https://medium.com/@robinpokorny/index-as-a-key-is-an-anti-pattern-e0349aece318

Tl;dr list items need to have a unique key that is not their list index